src/System.IO.Pipes/tests/AnonymousPipesThrowsTests.cs

```diff
@@ -27,17 +25,17 @@ public static void ServerBadPipeDirectionThrows()
         [Fact]
         public static void ServerBadInheritabilityThrows()
         {
-            Assert.Throws<ArgumentOutOfRangeException>(() =>new AnonymousPipeServerStream(PipeDirection.Out, (System.IO.HandleInheritability)999));
-            Assert.Throws<ArgumentOutOfRangeException>(() => new AnonymousPipeServerStream(PipeDirection.Out, (System.IO.HandleInheritability)999, 500));
+            Assert.Throws<ArgumentOutOfRangeException>("inheritability", () =>new AnonymousPipeServerStream(PipeDirection.Out, (System.IO.HandleInheritability)999));
+            Assert.Throws<ArgumentOutOfRangeException>("inheritability", () => new AnonymousPipeServerStream(PipeDirection.Out, (System.IO.HandleInheritability)999, 500));
         }
 
         [Fact]
         public static void ServerBadBufferThrows()
         {
-            Assert.Throws<ArgumentOutOfRangeException>(() => new AnonymousPipeServerStream(PipeDirection.Out, HandleInheritability.None, -500));
-            Assert.Throws<ArgumentOutOfRangeException>(() => new AnonymousPipeServerStream(PipeDirection.InOut, HandleInheritability.None, -500));
-            Assert.Throws<ArgumentOutOfRangeException>(() => new AnonymousPipeServerStream((PipeDirection)123, HandleInheritability.None, 500));
-            Assert.Throws<ArgumentOutOfRangeException>(() => new AnonymousPipeServerStream((PipeDirection)123, HandleInheritability.None, -500));
+            Assert.Throws<ArgumentOutOfRangeException>("bufferSize", () => new AnonymousPipeServerStream(PipeDirection.Out, HandleInheritability.None, -500));
+            Assert.Throws<ArgumentOutOfRangeException>("bufferSize", () => new AnonymousPipeServerStream(PipeDirection.InOut, HandleInheritability.None, -500));
```

Comment:
If we're doing both of these cases to verify the exception happens for both Out and InOut, should we also test for In?  Or should we consolidate it just down to one?